### ISSUE TITLE
Integrate xlsx accessibility checker for outputs?

### ISSUE THREAD

**Issue by matt-dray** [co-analysis/a11ytables#92]
Perhaps there could be some final flagging of the output xlsx to check for outstanding accessibility issues that need to be addressed.

Inspired by James Trimble's tool: [ObservableHQ](https://observablehq.com/@jtrim-ons/xlsx-accessibility-checker) and [standalone](https://github.com/jtrim-ons/xlsx-accessibility-checker).

(Also the chance here to create a standalone checker? Particularly re ODS, which James's tool can't do yet. Use [{tidyods}](https://github.com/mattkerlogue/tidyods), maybe?)

**Comment by matt-dray** [comment]
James's tool is now mentioned in the [resources for releasing statistics in spreadsheets](https://analysisfunction.civilservice.gov.uk/policy-store/further-resources-for-releasing-statistics-in-spreadsheets/) page of the Analysis Function website (as is {a11ytables}).

**Comment by matt-dray** [comment]
See also the DfT Shiny app: [department-for-transport-public/accessibility_checker](https://github.com/department-for-transport-public/accessibility_checker)

Given there are standalone options, I don't think this type of tool should be _integrated_ into {a11ytables}.
